### ISSUE TITLE
Error Login

### ISSUE THREAD

**Issue by WillFerdinando** [stylerx/OpenACARS#3]
When I click to login with the pilot id and password , nothing happens . I would like information

**Comment by stylerx** [comment]
You have the [PHPVMS Module](https://github.com/stylerx/OpenACARS-phpvms-module) correctly installed in your VA website?

**Comment by WillFerdinando** [comment]
Yes correct installed, you have installed version phpvms php 5.5 xx

**Comment by stylerx** [comment]
To fix the issue you can delete the OpenACARS folder in `%localappdata%` and rerun setup wizard **without** clicking on **test** button, and yes, i am Brazilian :+1:

**Comment by damonhb** [comment]
I'm getting a similar issue. It doesn't do a thing?

**Comment by stylerx** [comment]
@damonhb If you has tried the solution above, make sure that the entered URL doesn't contain a trailing slash.
